config/v1/types_apiserver.go

```diff
@@ -208,7 +214,7 @@ type APIServerEncryption struct {
 	// +openshift:enable:FeatureGate=KMSEncryptionProvider
 	// +unionMember
 	// +optional
-	KMS *KMSConfig `json:"kms,omitempty"`
+	KMS KMSConfig `json:"kms,omitzero,omitempty"`
 }
```

Comment:
Just wanting to verify - these changes are specifically because of linter issues that you were fixing?